src/components/Givers.js

```diff
@@ -65,15 +47,19 @@ class Givers extends Component {
             return this.renderNoStats(month - 1, +year)
         }
     }
-    render () {
+
+    render() {
         const {year, month} = this.props.match.params
         return (
-            <div>
-                <DateNavigation currentYear={year} currentMonth={month}/>
-                {
-                    this.state.loading ? <Spinner/> : this.renderStatsSection()
-                }
-            </div>
+            <>
```

Comment:
so this is equivalent to `React.Fragment`?

Reply:
Yes. It's a shorter syntax.